admin.go

```diff
@@ -167,6 +170,16 @@ func handleLoadConfig(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func handleStop(w http.ResponseWriter, r *http.Request) {
+	if r.Method == http.MethodPost {
+		log.Println("Shutting down...")
+		Stop()		
+		log.Println("Shut down.")
+		os.Exit(0)
```

Comment:
This line is the reason for the `empty response from server` message you're getting; the process is quitting before the server can reply to the API client. I suppose that's to be expected.

Reply:
Before calling os.Exit(), we absolutely need to be sure to call `certmagic.CleanUpOwnLocks()`. Maybe the steps to a graceful shutdown (it's really only two: Stop() and then that) should be put into their own function...

Reply:
I put these two in their own function